src/node/utils/cssUtils.ts

```diff
@@ -123,3 +124,44 @@ async function loadPostcssConfig(
     return (cachedPostcssConfig = null)
   }
 }
+
+export const cssImportMap = new Map<
+  string /*filePath*/,
+  Set<string /*filePath*/>
+>()
+
+export async function parseCssImport(
+  css: string,
+  filePath: string
+): Promise<string | Result> {
+  if (!css.includes('@import')) {
+    return css
+  }
+  return await postcss().use(cssImport()).process(css, { from: filePath })
+}
```

Comment:
This maybe can be combinate with `compileStyleAsync` use `postcss-easy-import ` plugin(I meet some error with this). And it need `compiler-sfc` export `postcss` messages.
So I only implement this...